packages/web-components/src/menu-item/menu-item.ts

```diff
@@ -236,47 +180,37 @@ export class MenuItem extends FASTElement {
    * @internal
    */
   public handleMouseOver = (e: MouseEvent): boolean => {
-    if (this.disabled || !this.hasSubmenu || this.expanded) {
-      return false;
-    }
-
-    this.expanded = true;
+    if (this.disabled) return false;
 
+    this.submenu?.togglePopover(true);
     return false;
   };
 
   /**
    * @internal
    */
   public handleMouseOut = (e: MouseEvent): boolean => {
-    if (!this.expanded || this.contains(document.activeElement)) {
+    if (this.contains(document.activeElement)) {
       return false;
     }
 
-    this.expanded = false;
+    this.submenu?.togglePopover(false);
```

Comment:
This is more of a design question. Do we want to add a delay before closing the submenu? The delay will allow the cursor to cross from the menu item to a long submenu if needed.